torchtune/modules/loss/kd_losses.py

```diff
@@ -138,3 +237,164 @@ def forward(
             )
 
         return total_fkl_loss / torch.sum(mask.view(-1), dim=0)
+
+class ReverseKLWithChunkedOutputLoss(torch.nn.Module):
```

Comment:
Not necessary for this PR but as we are starting to have a proliferation of chunked loss implementations I wonder whether it'd be worth investing in a general utility to wrap an arbitrary loss with chunking operation @felipemello1